extensions/xslt/runtime/src/main/java/org/apache/camel/quarkus/component/xslt/RuntimeUriResolver.java

```diff
@@ -60,11 +60,7 @@ public Source resolve(String href, String base) throws TransformerException {
      * @return     the unqualified translet name associated with the given {@code uri}
```

Comment:
Just guessing - please correct me if needed:

```suggestion
     * @return     the unqualified translet name associated with the given {@code uri} or {@code null} if the given XSLT resource was not compiled to a translet at build time.
```